explore/src/main/scala/explore/observationtree/TreeComp.scala

```diff
@@ -1,172 +1,172 @@
 // Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
We are not using this component, but we will probably use it in the future, that's why I'm leaving it commented out for the moment. Migration to reusable functions is not trivial in this case.